README.md

```diff
@@ -1,7 +1,108 @@
+                'modeling_end_time': datetime.datetime(2010, 1, 16, 0, 0),
+                'train_matrix': {
+                    'matrix_start_time': datetime.datetime(2010, 1, 1, 0, 0),
+                    'matrix_end_time': datetime.datetime(2010, 1, 6, 0, 0),
+                    'as_of_times': [
+                        datetime.datetime(2010, 1, 1, 0, 0),
+                        datetime.datetime(2010, 1, 2, 0, 0),
+                        datetime.datetime(2010, 1, 3, 0, 0),
+                        datetime.datetime(2010, 1, 4, 0, 0),
+                        datetime.datetime(2010, 1, 5, 0, 0)
+                    ],
+                    'label_window': '1 day',
+                    'example_frequency': '1 days',
+                    'train_duration': '5 days'
+                },
+                'test_matrices': [{
+                    'matrix_start_time': datetime.datetime(2010, 1, 6, 0, 0),
+                    'matrix_end_time': datetime.datetime(2010, 1, 11, 0, 0),
+                    'as_of_times': [
+                        datetime.datetime(2010, 1, 6, 0, 0),
+                        datetime.datetime(2010, 1, 9, 0, 0),
+                    ],
+                    'label_window': '3 months',
+                    'example_frequency': '3 days'
+                }]
+            },
+            {
+                'beginning_of_time': datetime.datetime(1990, 1, 1, 0, 0),
+                'modeling_start_time': datetime.datetime(2010, 1, 1, 0, 0),
+                'modeling_end_time': datetime.datetime(2010, 1, 16, 0, 0),
+                'train_matrix': {
+                    'matrix_start_time': datetime.datetime(2010, 1, 6, 0, 0),
+                    'matrix_end_time': datetime.datetime(2010, 1, 11, 0, 0),
+                    'as_of_times': [
+                        datetime.datetime(2010, 1, 6, 0, 0),
+                        datetime.datetime(2010, 1, 7, 0, 0),
+                        datetime.datetime(2010, 1, 8, 0, 0),
+                        datetime.datetime(2010, 1, 9, 0, 0),
+                        datetime.datetime(2010, 1, 10, 0, 0)
+                    ],
+                    'label_window': '1 day',
+                    'example_frequency': '1 days',
+                    'train_duration': '5 days'
+                },
+                'test_matrices': [{
+                    'matrix_start_time': datetime.datetime(2010, 1, 11, 0, 0),
+                    'matrix_end_time': datetime.datetime(2010, 1, 16, 0, 0),
+                    'as_of_times': [
+                        datetime.datetime(2010, 1, 11, 0, 0),
+                        datetime.datetime(2010, 1, 14, 0, 0)
+                    ],
+                    'label_window': '3 months',
+                    'example_frequency': '3 days'
+                }]
+            }
+        ]
+```
+
+Timechop currently works with the following:
+
+- `beginning_of_time` - earliest date included in features
+- `modeling_start_time` - earliest date in any model
```

Comment:
I think we want to be clearer here. Maybe "earliest date at which features are calculated"?